bin/build-locales.js

```diff
@@ -0,0 +1,18 @@
+#! /usr/bin/env node
+
+const spawn = require('cross-spawn');
+
+let locales = '*';
+if (process.env.NODE_ENV === 'production' &&
+    process.env.npm_package_config_productionLocales) {
+  locales = process.env.npm_package_config_productionLocales;
+}
+
+spawn.sync('pontoon-to-webext', [], {
+  stdio: 'inherit',
+  env: Object.assign(
+    {},
+    process.env,
+    { SUPPORTED_LOCALES: locales }
+  )
+});
```

Comment:
Yeah, I figured this was more a Snooze Tabs specific hack for npm scripts switching on NODE_ENV==production and using the package.json config. Other projects might want to do this differently (e.g. if they're using gulp or webpack)